src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -118,7 +155,31 @@ else if (this.performance == null || Entreprise.PERFORMANCE_BASE.equals(this.per
     }
 
     //Augmenter salaire
-    //public void augmenterSalaire(double pourcentage){}
+    /**
+     * Calcul de l'augmentation du salaire d'un employé.
+     *
+     * Cette méthode augmente tout simplement le salaire de l'employé selon un pourcentage passé en paramètres.
+     *
+     * Si le pourcentage passé en paramètre est négatif ou égal à 0, on retourne le salaire de base sans augmentation
+     * Si le salaire de l'employé est null ou qu'il est inférieur au salaire de base de l'entreprise, on attribue le salaire de base de l'entreprise à l'employé
+     *
+     * Si le pourcentage et le salaire sont ok, on fait une augmentation du salaire en fonction du pourcentage passé
+     * on fait en sorte d'arrondir le salaire pour ne garder que deux chiffres après la virgule et on retourne le salaire
+     *
+     * @param pourcentage le pourcentage d'augmentation du salaire de l'employe
+     *
+     */
+    public double augmenterSalaire(double pourcentage){
+        //
+        if(pourcentage <= 0){
+            return this.getSalaire();
+        } else if (this.getSalaire() == null || this.getSalaire() <= Entreprise.SALAIRE_BASE){
+            this.setSalaire(Entreprise.SALAIRE_BASE);
+        }
+        this.salaire = this.getSalaire()+(this.getSalaire()*(pourcentage/100d));
+        this.salaire = (double) Math.round(this.salaire*100)/100;
```

Comment:
Tu pourrai ensuite faire ton arrondi (sans faire le cast) en fesant :
 this.salaire =  Math.round(nouveauSalaire * 100) / 100d; 
 
 le (d) fais la conversion pour toi directement en double ( pour simplifier l'idée)